modules/custom/cp_station_list/cp_station_list.libraries.yml

```diff
@@ -9,6 +9,7 @@ station-list:
     //static.icos-cp.eu/constant/misc/countries.js: { type: external, minified: true }
     //cdn.datatables.net/1.10.22/js/jquery.dataTables.js: { type: external, minified: true }
     //cdn.datatables.net/1.10.22/js/dataTables.bootstrap4.js: { type: external, minified: true }
+    https://static.icos-cp.eu/share/stationscommon/stations.js: { type: module, minified: false }
```

Comment:
Does `type: module` exist? I would have used `external` like for the other files.